src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -14138,16 +14116,14 @@ DECLARE_API(SaveState)
     ExtOut("Session breakpoints and watch expressions saved to %s\n", filePath.data);
     return S_OK;
 }
-#endif
 
+#endif // FEATURE_PAL
```

Comment:
The ifdef at line 13945.  This endif was moved from  line 14170 (original file).